sigs.yaml

```diff
@@ -3511,9 +3511,9 @@ workinggroups:
       name: Yuan Tang
       company: Red Hat
   meetings:
-  - description: WG Serving weekly meeting ([Calendar](https://calendar.google.com/calendar/embed?src=e896b769743f3877edfab2d4c6a14132b2aa53287021e9bbf113cab676da54ba%40group.calendar.google.com))
+  - description: WG Serving Weekly Meeting
```

Comment:
let's keep calendar link.

Reply:
++

Reply:
I agree that this would increase visibility. Wondering if there's a way to fix the formatting.

We also mention the calendar link in the meeting notes and encourage people joining the mailing list.

Reply:
Looks like this is worth revisiting: https://github.com/kubernetes/community/blob/master/generator/wg_readme.tmpl#L22

Reply:
I found the issue. Please take another look.